exporter/debugexporter/README.md

```diff
@@ -60,8 +60,22 @@ Here's an example output:
 
 ### Normal verbosity
 
-With `verbosity: normal`, the exporter's behavior is currently the same as with `verbosity: basic`.
-See above for more details.
+With `verbosity: normal`, the exporter outputs about one line for each telemetry record.
```

Comment:
Should we document which LogRecord fields will be included into the log line?

```suggestion
With `verbosity: normal`, the exporter outputs about one line for each telemetry record, including its body and attributes.
```

Reply:
Yes, this is probably useful. :+1: